cli.js

```diff
@@ -28,30 +28,33 @@ function start (opts) {
 
   if (opts.help) {
     console.log(fs.readFileSync(path.join(__dirname, 'help.txt'), 'utf8'))
-    stop(0)
+    return module.exports.stop()
   }
 
   assert(opts._.length === 1, 'Missing the file parameter')
 
   return runFastify(opts)
 }
 
-function stop (code) {
-  process.exit(code)
+function stop (error) {
+  if (error) {
+    console.log(error)
+    process.exit(1)
+  }
+  process.exit()
 }
 
 function runFastify (opts) {
   var file = null
   try {
     file = require(path.resolve(process.cwd(), opts._[0]))
   } catch (e) {
-    console.log(`Cannot find the specified file: '${opts._[0]}'`)
-    stop(1)
+    return module.exports.stop(e)
```

Comment:
Why?

Reply:
Because I can ovewrite it during tests

Reply:
I don't like so much as solution, is not elegant. But for the moment could be ok. :)

Reply:
Me too. But I have not sadly found another solution!